extension/fts/test/test_files/fts_small.test

```diff
@@ -283,3 +283,38 @@ Binder exception: Table: 0_docIdx3_appears_info already exists. Please drop or r
 -STATEMENT CALL CREATE_FTS_INDEX('doc', 'docIdx4', ['content'])
 ---- error
 Binder exception: Table: 0_docIdx4_appears_in already exists. Please drop or rename the table before creating a full text search index.
+
+-CASE show_indexes
+-SKIP_IN_MEM
```

Comment:
Why are we skipping IN_MEM?

Reply:
reloaddb serialize/deserialize the indexentry to disk